src/Mutator/Number/DecrementInteger.php

```diff
@@ -159,4 +165,34 @@ private function isArrayZeroIndexAccess(Node $node): bool
 
         return false;
     }
+
+    /**
+     * @param Node\Scalar\LNumber $node
+     */
+    private function isPregSplitLimitZeroValue(Node $node): bool
```

Comment:
I had to dig manual to understand what this function does. It might need a better name.

Reply:
`isLimitArgWithZeroValueOfPregSplitFunction`  ?